skore/src/skore/sklearn/cross_validation/plots/compare_scores_plot.py

```diff
@@ -0,0 +1,118 @@
+"""Plot cross-validation results."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import plotly.graph_objects
+
+
+def plot_cross_validation_compare_scores(
+    cv_results: dict,
+) -> plotly.graph_objects.Figure:
+    """Plot the result of a cross-validation run.
+
+    Parameters
+    ----------
+    cv_results : dict
+        The output of scikit-learn's cross_validate function.
+
+    Returns
+    -------
+    plotly.graph_objects.Figure
+        A plot of the cross-validation results, without the time-related results.
+    """
+    import pandas
+    import plotly
+    import plotly.graph_objects as go
+
+    _cv_results = cv_results.copy()
+
+    # Remove irrelevant keys
+    to_remove = ["indices", "estimator"] + [
+        key
+        for key in _cv_results
+        if key.startswith("fit_time") or key.startswith("score_time")
+    ]
+    for key in to_remove:
+        _cv_results.pop(key, None)
+
+    df = pandas.DataFrame(_cv_results)
+
+    if "test_score" in df.columns:
+        df.insert(0, "test_score", df.pop("test_score"))
+
+    def linspace(lo, hi, num):
+        interval = (hi - lo) / (num - 1)
+        return [lo + k * interval for k in range(0, num)]
```

Comment:
Maybe you can use [`numpy.linspace`](https://numpy.org/doc/2.1/reference/generated/numpy.linspace.html) here ?

Reply:
I did originally, but re-implementing it avoids bringing in numpy as a direct dependency for just 1 function

Reply:
...although I see that we import numpy in `cross_validation_item.py`...

Reply:
Done, added numpy to our direct deps